packages/app/navigation/desktop/MessagesNavigator.tsx

```diff
@@ -0,0 +1,163 @@
+import {
+  DrawerContentComponentProps,
+  createDrawerNavigator,
+} from '@react-navigation/drawer';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { NavigationState } from '@react-navigation/routers';
+import { View, getVariableValue, useTheme } from 'tamagui';
+
+import { ChannelMembersScreen } from '../../features/channels/ChannelMembersScreen';
+import { ChannelMetaScreen } from '../../features/channels/ChannelMetaScreen';
+import { EditProfileScreen } from '../../features/settings/EditProfileScreen';
+import ChannelScreen from '../../features/top/ChannelScreen';
+import ChannelSearchScreen from '../../features/top/ChannelSearchScreen';
+import { ChatDetailsScreen } from '../../features/top/ChatDetailsScreen';
+import { ChatListScreenView } from '../../features/top/ChatListScreen';
+import { ChatVolumeScreen } from '../../features/top/ChatVolumeScreen';
+import { GroupChannelsScreenContent } from '../../features/top/GroupChannelsScreen';
+import ImageViewerScreen from '../../features/top/ImageViewerScreen';
+import PostScreen from '../../features/top/PostScreen';
+import { UserProfileScreen } from '../../features/top/UserProfileScreen';
+import { GroupSettingsStack } from '../GroupSettingsStack';
+import { HomeDrawerParamList } from '../types';
+
+const HomeDrawer = createDrawerNavigator();
+
+export const HomeNavigator = () => {
+  return (
+    <HomeDrawer.Navigator
+      drawerContent={DrawerContent}
+      initialRouteName="ChatList"
+      screenOptions={{
+        drawerType: 'permanent',
+        headerShown: false,
+        drawerStyle: {
+          width: 400,
```

Comment:
We should probably make the width 0 when we're navigated to the image viewer here too.

Reply:
will do this in the follow-up since we aren't using this yet